### ISSUE TITLE
<!DOCTYPE html>

### ISSUE THREAD

**Issue by GeberID** [GeberID/Web#1]
<html>

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <title>No! official site</title>
    <link href="https://fonts.googleapis.com/css?family=Montserrat+Alternates:400&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Jura&display=swap&subset=cyrillic" rel="stylesheet">
    <link rel="stylesheet" href="style/normalize.css" />
    
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
    <link rel="stylesheet" media="screen and (max-width: 544px)" href="/style/menu.css" />
    <link rel="stylesheet" href="style/main.css" />
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    
    <script src="https://cdnjs.cloudflare.co
